enginetest/enginetests.go

```diff
@@ -2425,6 +2425,20 @@ func TestDropColumn(t *testing.T, harness Harness) {
 	})
 }
 
+func TestAddAndDropColumn(t *testing.T, harness Harness) {
```

Comment:
You should build Dolt against this PR and make sure dolt passes this pr.

Reply:
Also this is a separate test case all together. Look at `dolt_engine_test` to see how to run this test against Dolt

Reply:
I'm also fine with this just being a BATs test case. And a simpler AddColumn test in gms